runtime/near-vm-runner/src/tests.rs

```diff
@@ -55,34 +55,45 @@ fn create_context(input: Vec<u8>) -> VMContext {
     }
 }
 
-fn make_simple_contract_call_with_gas_vm(
+/// Exhaustive parameters for making a simple contract call.
+/// Calling with no protocol version specified will use a test configuration.
+fn make_simple_contract_call_ex(
     code: &[u8],
     method_name: &str,
-    prepaid_gas: u64,
     vm_kind: VMKind,
+    protocol_version: Option<ProtocolVersion>,
+    prepaid_gas: u64,
 ) -> (VMOutcome, Option<VMError>) {
     let mut fake_external = MockedExternal::new();
     let mut context = create_context(vec![]);
     context.prepaid_gas = prepaid_gas;
-    let config = VMConfig::test();
-    let fees = RuntimeFeesConfig::test();
+    let (config, fees) = if let Some(protocol_version) = protocol_version {
+        let runtime_config_store = RuntimeConfigStore::new(None);
+        let runtime_config = runtime_config_store.get_config(protocol_version);
+        (runtime_config.wasm_config.clone(), runtime_config.transaction_costs.clone())
+    } else {
+        (VMConfig::test(), RuntimeFeesConfig::test())
+    };
 
     let promise_results = vec![];
-
     let code = ContractCode::new(code.to_vec(), None);
-    let runtime = vm_kind.runtime(config).expect("runtime has not been compiled");
-    runtime
-        .run(
-            &code,
-            method_name,
-            &mut fake_external,
-            context,
-            &fees,
-            &promise_results,
-            LATEST_PROTOCOL_VERSION,
-            None,
-        )
-        .outcome_error()
+
+    if let Some(runtime) = vm_kind.runtime(config) {
```

Comment:
nit: Should we consider keeping the previous style namely because it has fewer indentations?